packages/paypal-commerce-utils/src/paypal-commerce-types.ts

```diff
@@ -37,15 +37,27 @@ export interface PayPalCommerceInitializationData {
     // paymentButtonStyles?: Record<string, PayPalButtonStyleOptions>; // TODO: PayPalButtonStyleOptions interface will be moved in the future
 }
 
+/**
+ *
+ * PayPal Commerce Sdk Namespaces
+ *
+ */
+export enum PayPalCommerceSdkNamespaces {
+    PaypalAxo = 'paypalAxo',
+    PaypalMessages = 'paypalMessages',
+    PaypalConnect = 'paypalConnect',
+}
```

Comment:
This is not quite right to set `paypalConnect` as a namespace because this is a part of PayPalAxo implementation, but not a part of PP SDK initialization.

Reply:
But we add it as the value of the `data-namespace` attribute in the `script`

Reply:
You are right. I confused it with `paypalAxo`